SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -176,13 +181,13 @@ final class SwedbankPaySDKViewModel: NSObject {
             
             let json = try! JSONEncoder().encode(["paymentorder": paymentOrder])
             
-            var request = try! URLRequest(url: endPointUrl, method: .post, headers: self?.configuration?.headers)
+            var request = try! URLRequest(url: endPointUrl, method: .post, headers: self?.configuration?.afHeaders)
             if request.value(forHTTPHeaderField: "Content-Type") == nil {
                 request.setValue("application/json", forHTTPHeaderField: "Content-Type")
             }
             request.httpBody = json
             
-            self?.sessionManager.request(request).responseJSON(completionHandler: { [weak self] response in
+            self?.sessionManager.request(request).responseDecodable(of: OperationsList.self, completionHandler: { [weak self] response in
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 137 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)